app/apps/server/bridges/http.js

```diff
@@ -1,15 +1,23 @@
 import { HTTP } from 'meteor/http';
 
 export class AppHttpBridge {
+	constructor(orch) {
+		this.orch = orch;
+	}
+
 	async call(info) {
+		if (typeof info.request.timeout !== 'number' || info.request.timeout > 500) {
+			info.request.timeout = 500;
```

Comment:
I'm not sure 500 is a good number. Wondering how many requests may start failing after adding this timeout.

also because of this I think we should tag as a break change.

Reply:
The VM has a timeout of 1000ms, that's why 500ms.

I think it's more a fix than a break change, since the HTTP not respecting the VM's timeout wasn't expected.